strawberry/types/fields/resolver.py

```diff
@@ -68,17 +68,23 @@ class ReservedName(NamedTuple):
     name: str
 
     def find(
-        self, parameters: Tuple[inspect.Parameter, ...], _: StrawberryResolver
+        self,
+        parameters: Tuple[inspect.Parameter, ...],
+        resolver: StrawberryResolver[Any],
     ) -> Optional[inspect.Parameter]:
+        del resolver
```

Comment:
why? :D

Reply:
To satisfy Pyright's hatred of unused arguments 😂 It's the suggested way by PyLint to
fix unused argument warnings / errors. Can't use `_` here as it violates Liskov
since the signature of find would be different 😥